src/context.ts

```diff
@@ -30,7 +31,8 @@ export async function getInputs(): Promise<Inputs> {
     vtMonitor: core.getBooleanInput('vt_monitor'),
     monitorPath: core.getInput('monitor_path') || '/',
     updateReleaseBody: core.getBooleanInput('update_release_body'),
-    githubToken: core.getInput('github_token')
+    githubToken: core.getInput('github_token'),
+    requestRate: parseInt(core.getInput('request_rate'))
```

Comment:
Define default here too, just in case the action is invoked outside a GitHub Runner:

```suggestion
    requestRate: parseInt(core.getInput('request_rate') || '1000')
```